### PR TITLE
Feature allow specifying network

### PR DESCRIPTION
**Pull request by champly**
### Issues associated with this PR

#1389

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases. And you need to show the UT result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result.

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/types/xds_test.go [hunk 1]
```diff
@@ -41,7 +42,7 @@ func TestInitXdsFlags(t *testing.T) {
 		"Not_exist_key:1",
 		"not_exist_value",
 	}, []string{})
-	if !assert.Equal(t, 3, len(xdsInfo.Metadata.GetFields()), "serviceMeta len should be one") {
+	if !assert.Equal(t, 4, len(xdsInfo.Metadata.GetFields()), "serviceMeta len should be one") {
 		t.FailNow()
 	}
 	if !assert.Equal(t, "1.1", xdsInfo.Metadata.Fields["ISTIO_VERSION"].GetStringValue(), "serviceMeta len should be one") {
```
Comment (wangfakang) [inline pkg/types/xds_test.go:48]: could add test case about check `ClusterID`.

**Comment by champly** [comment]
It looks like a Istio prob with this [pr](https://github.com/istio/istio/pull/28329/files)

**Comment by champly** [comment]
fix with [pr](https://github.com/istio/istio/pull/28329)

**Review by wangfakang — APPROVED** [review]
LGTM.
There is a problem with the version between istio-1.6 and  istio-1.8. fixed by https://github.com/istio/istio/pull/28329

**Pull request merged**